frontend/src/components/common/Catalog/Catalog.module.scss

```diff
@@ -0,0 +1,96 @@
+@import '../../../variables';
+
+.categoryWrapper {
+  display: flex;
+  position: absolute;
+  z-index: 9999;
+  top: 75px;
+  left: 0;
+
+  .itemsWrapper {
```

Comment:
All of our classes are unique, you don't need to dump everything into your parent's class